tests/test_csp_sha.py

```diff
@@ -0,0 +1,27 @@
+import base64
+import hashlib
+
+from bs4 import BeautifulSoup
+
+from tests.src import env
+
+
+def test_csp_sha():
```

Comment:
we could name it `test_content_security_policy_sha`, both in filename and test name, this way it will be more readable and provide additional context for a person that reads it for the first time, or first time in a long while ☺️

Reply:
Good point 💯. I've made those changes in https://github.com/alphagov/notifications-govuk-alerts/pull/111/commits/971ff6044dd0e8a86e90bf5aa4cbe2a27817051f.